This gets applied in libraries/source/nvtt/nvtt-${_nvtt_version}
--- a/src/src/nvmath/SimdVector_VE.h
+++ b/src/src/nvmath/SimdVector_VE.h
@@ -27,10 +27,7 @@
 #ifndef NV_SIMD_VECTOR_VE_H
 #define NV_SIMD_VECTOR_VE_H
 
-#ifndef __APPLE_ALTIVEC__
 #include <altivec.h>
-#undef bool
-#endif
 
 namespace nv {
 
diff --git a/src/src/nvtt/squish/simd_ve.h b/src/src/nvtt/squish/simd_ve.h
index 5a11858..e256c54 100644
--- a/src/src/nvtt/squish/simd_ve.h
+++ b/src/src/nvtt/squish/simd_ve.h
@@ -27,10 +27,7 @@
 #ifndef SQUISH_SIMD_VE_H
 #define SQUISH_SIMD_VE_H
 
-#ifndef __APPLE_ALTIVEC__
 #include <altivec.h>
-#undef bool
-#endif
 
 namespace nvsquish {
 
